.github/workflows/docker.yml

```diff
@@ -15,6 +15,9 @@ jobs:
       -
         name: Check out
         uses: actions/checkout@v3
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
```

Comment:
Nit: I don't know how expensive this step is, so I'd prefer to avoid running it if tests are going to fail. (Please move it down to just before the second build step so that it doesn't run if tests fail.)

Reply:
Also add:

```yaml
        if: github.event_name == 'push'
```

because there's no point in setting up QEMU if it's not going to be used.